src/interface.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+$(document).ready(function() {
+  var player = new Player(`Player`);
```

Comment:
Good idea to add a player object! Makes it more personal and you could add a scoreboard feature in the future